CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Removes `_max_timeout` and instead uses a flat 60 second timeout for requests
 * Adds Python version to user-agent header on requests
 * Removes `shipment.get_rates()` method since the shipment object already has rates. If you need to get new rates for a shipment, please use the `shipment.regenerate_rates()` method.
+* Add `RetrieveMe()` Convenience Function that allow users to retrieve without specifying an ID.
```

Comment:
When referencing a function, you need to reference it correctly for the language you are working in.
```suggestion
* Add `retrieve_me()` convenience function that allow users to retrieve without specifying an ID.
```